python-avd/tests/pyavd/j2filters/test_status_render.py

```diff
@@ -0,0 +1,16 @@
+# Copyright (c) 2023-2024 Arista Networks, Inc.
+# Use of this source code is governed by the Apache License 2.0
+# that can be found in the LICENSE file.
+from __future__ import annotations
+
+import pytest
+from pyavd.j2filters.status_render import status_render
+
+STATE_STRINGS = [("PASS", "github", ":white_check_mark:"), ("FAIL", "test", "FAIL")]
```

Comment:
let's add a test to valdiate the `upper` part of the Github code
```suggestion
STATE_STRINGS = [("PASS", "github", ":white_check_mark:"), ("fail", "github", ":x:"), ("FAIL", "test", "FAIL")]
```